Testing.md

```diff
@@ -70,7 +70,11 @@ If you want to run code style verification only for one tool, use `bundle exec r
 
 After introducing some changes to the _fastlane_ source code, you probably want to test the changes for your application. The easiest way to do so it use [bundler](https://bundler.io/).
 
-Copy the `Gemfile` [.assets/Gemfile](.assets/Gemfile) from your local _fastlane_ clone and drop it into your project's root folder. If you already have a `Gemfile` in your project's root folder, adapt it to match the template.
+Adapt your `Gemfile` in your project's root folder and replace the `gem 'fastlane'` line with:
```

Comment:
Can we change `Adapt` to `Edit`? `Edit` feels like it goes better with `replace` later on in this line 🙃

Reply:
Sure.

I hereby give you permission to make nitpicky changes in my PRs yourself before merging, especially if they only concern one word 🍅